openedx-unsupported/configuration
# [SE-4481] Add feature to sync edx services logs

This PR reintroduces the feature that was included in #6418 but had to be reverted via #6464, by putting it behind a boolean flag named COMMON_OBJECT_STORE_EDX_LOG_SYNC. The flag is disabled by default so as to not change the existing behaviour.

**JIRA Ticket:** [SE-4481](https://tasks.opencraft.com/browse/SE-4481)

**Sandbox URL:** TBD

**Testing Instructions:**

- Set the `COMMON_OBJECT_STORE_EDX_LOG_SYNC` to true and the required prefix for `COMMON_OBJECT_STORE_EDX_LOG_SYNC_PREFIX`.
- Launch new instance and verify that the edx-services logs are uploaded to the configured object storage when the logrotates.
